src/Models/WebMapServiceCatalogGroup.js

```diff
@@ -59,6 +59,12 @@ var WebMapServiceCatalogGroup = function(application) {
      */
     this.blacklist = undefined;
 
+    /**
+     * Specifies whether to use each WMS layer's "title" (default), "name", or "abstract" as the primary title.
+     * @type {String}
+     */
+    this.titlefield = 'title';
+
     knockout.track(this, ['url', 'dataCustodian', 'parameters', 'blacklist']);
```

Comment:
'titleField` should be added her as well so that it becomes a knockout observable and we can subscribe to its change notification.

It should also be added to the list of properties returned by `_getValuesThatInfluenceLoad` because it does.